### PR TITLE
Add active flag to complexities

### PR DESCRIPTION
**Pull request by doctorpod**
Ticket https://dsdmoj.atlassian.net/browse/MO-676

### CODE REVIEW

File: spec/requests/v1/complexities_request_spec.rb [hunk 1]
```diff
@@ -24,21 +24,29 @@
...
+      expired_token: expired_token
+    )
   end
 
   describe "GET /v1/complexity-of-need/offender-no/:offender_no" do
     let(:endpoint) { "/v1/complexity-of-need/offender-no/#{offender_no}" }
     let(:offender_no) { complexity.offender_no }
     let!(:complexity) { create(:complexity) }
-
+
```
Comment (SebNegahbanMoJ) [inline spec/requests/v1/complexities_request_spec.rb]: _Extremely_ minor thing of unnecessary whitespace. If you have to go back in to change anything else and remember that it's there then it may be worth deleting, otherwise it's no biggie!

**Review by SebNegahbanMoJ — APPROVED** [review]

File: spec/requests/v1/complexities_request_spec.rb (hunk 1 above)
Comment (ayqazi) [reply spec/requests/v1/complexities_request_spec.rb]: @doctorpod can you configure your editor to delete extraneous whitespace from the end of lines and end of files?

**Comment by ayqazi** [comment]
I don't understand the purpose of this service so @SebNegahbanMoJ is the best placed to review it (and he has!)

**Comment by SebNegahbanMoJ** [comment]
> I don't understand the purpose of this service so @SebNegahbanMoJ is the best placed to review it (and he has!)

Full disclosure, I'm not familiar with the service either, but given that the goal was to give a model an 'inactive' flag and make it available via an API, it seemed like the changes were reasonable

**Review by mgreen-mojhub — APPROVED** [review]
LGTM
